### PR TITLE
[SYCL][ESIMD] Enlarge stateless stack size for tests with private memory

### PR DESCRIPTION
**Pull request by fveselov**

### CODE REVIEW

**Comment by DenisBakhvalov** [comment]
Please update the PR title.

**Comment by DenisBakhvalov** [comment]
nit: I would better name it like:
"[ESIMD] Enlarge stateless stack size for tests with private memory"

File: SYCL/ESIMD/private_memory/pm_access_1.cpp [hunk 1]
```diff
@@ -7,7 +7,6 @@
 //===----------------------------------------------------------------------===//
 
 // REQUIRES: gpu
-// Sporadic failure in GPU RT 21.16.19610
-// UNSUPPORTED: cuda || (level_zero && linux) || (opencl && linux)
-// RUN: %clangxx -fsycl -I%S/.. %S/Inputs/pm_common.cpp -o %t.out
+// UNSUPPORTED: cuda
+// RUN: %clangxx -fsycl -Xs "-stateless-stack-mem-size=131072" -I%S/.. %S/Inputs/pm_common.cpp -o %t.out
```
Comment (vladimirlaz) [inline SYCL/ESIMD/private_memory/pm_access_1.cpp:11]: Will the options work is CL-style clang is used (aka lang-cl)?
If not mark them UNSUPPORTED: cl_options

Comment (fveselov) [reply SYCL/ESIMD/private_memory/pm_access_1.cpp:11]: I believe option should work fine when CL-style clang is used. Jenkins/pre-ci-windows successfully passed, so I think marking unsupported is not needed.

**Review by vladimirlaz — APPROVED** [review]

**Review by DenisBakhvalov — APPROVED** [review]

**Comment by v-klochkov** [comment]
@fveselov, @vladimirlaz, @DenisBakhvalov, @kbobrovs - The test pm_access_2.cpp started failing on unrelated changes:
1) http://icl-jenkins.sc.intel.com:8080/blue/organizations/jenkins/SYCL_CI%2Fintel%2FLin%2FLLVM_Test_Suite/detail/LLVM_Test_Suite/5168/pipeline/311
2) http://icl-jenkins.sc.intel.com:8080/blue/organizations/jenkins/SYCL_CI%2Fintel%2FLin%2FLLVM_Test_Suite/detail/LLVM_Test_Suite/5174/pipeline

**Comment by kbobrovs** [comment]
@fveselov, please mark these tests and those below as UNSPPORTED until this is fully resolved. Problem with these tests pop-up regularly during testing. One more case: http://icl-jenkins.sc.intel.com:8080/blue/organizations/jenkins/SYCL_CI%2Fintel%2FLin%2FLLVM_Test_Suite/detail/LLVM_Test_Suite/5176/pipeline
\+ @bader

**Comment by fveselov** [comment]
[PR](https://github.com/intel/llvm-test-suite/pull/318)

**Pull request merged**